camel-k-runtime-core/src/generated/resources/META-INF/services/org/apache/camel/configurer/SourceDefinition

```diff
@@ -0,0 +1,2 @@
+# Generated by camel build tools - do NOT edit this file!
+class=org.apache.camel.k.SourceDefinitionConfigurer
```

Comment:
camel now generate these via FQN name, so just beware of deleting this old and adding the new

Reply:
I'm on it